docs/docs/configuration-file.md

```diff
@@ -74,10 +74,10 @@ theme = "dracula"
 
 ## Padding-x
 
-Define x axis padding (default is 10)
+Define x axis padding (default is 0)
 
 ```toml
-padding-x = 10
```

Comment:
I would like to keep the bottom example (so people can have an idea how the updates are done) but the `(default is 10)` needs to be fixed to `0`

Reply:
Makes sense, i'll fix it